providers/unleash/pom.xml

```diff
@@ -20,7 +20,7 @@
         <dependency>
             <groupId>io.getunleash</groupId>
             <artifactId>unleash-client-java</artifactId>
-            <version>8.4.0</version>
+            <version>9.0.0</version>
```

Comment:
@liran2000 I was wondering about this.

Is it better to consider this a provided dependency and give it a range? So consumers have more control? There are pros/cons to both choices. I'm wondering if you considered them.

Reply:
My preference is a specific version for consistent builds.
consumers have control, they can override the version in their POM.